projects/packages/search/src/customberg/components/sidebar/styles.scss

```
@@ -57,7 +59,7 @@
 	&.is-active {
 		// The transparent shadow ensures no jumpiness when focus animates on an active tab.
 		box-shadow: inset 0 0 0 var( --wp-admin-border-width-focus ) transparent,
-			inset 0 0 -$border-width-tab 0 0 var( --wp-admin-theme-color );
+			inset 0 0 -$border-width-tab 0 0 var( --wp-admin-theme-color ); // stylelint-disable-line scss/operator-no-unspaced -- this is a sign, not an operation
```

As far as I can tell, scss really is interpreting it as an operation. The output CSS for this rule is
```
box-shadow:inset 0 0 0 var(--wp-admin-border-width-focus) #0000,inset 0 -1.5px 0 0 #3858e9;
```
And further I note that `inset 0 0 -1.5px 0 0 #3858e9` would actually be invalid.

My best guess is that back in #20369 they noticed one of the 0s was disappearing, added an extra one, and moved on.

Looks like this will work and be less confusing.
```suggestion
			inset 0 (-$border-width-tab) 0 0 var( --wp-admin-theme-color );

---

Ah, good catch.